spec/diddocuments.md

```diff
@@ -184,8 +185,100 @@ Secp256k1 public keys must be converted to a verification method with a type of
 
 TODO: Add example of Secp256k1 key in a KEL and the resultant verification method
 
+##### Thresholds
+If the current signing keys threshold (the value of the `kt` field) is greater than 1, then in addition to the verification
```

Comment:
I think the multi-sig description should mostly reference the `threshold` value (since it is in the DIDDoc), with a possible reference to say that the `threshold` is set by the `kt` value from the KERI state.  Later references to the value should to the `threshold`.

Reply:
You are right that `threshold` is set by the `kt` value, but it will only even be in the DID document if `kt` > 1, so that's why this sentence references `kt`.